hyperspy/_components/gaussian.py

```diff
@@ -209,3 +212,15 @@ def fwhm(self):
     @fwhm.setter
     def fwhm(self, value):
         self.sigma.value = value / sigma2fwhm
+
+    def array(self, axis):
```

Comment:
Wouldn't it be better to make this function private since it is not available for all components? Also, wouldn't ``nd_function`` or ``function_nd`` be a more explicit name?